### PR TITLE
Test/ci

### PR DESCRIPTION
**Pull request by zcgandcomp**

### CODE REVIEW

**Review by romanstrobl — CHANGES_REQUESTED** [review]
I prefer not to include the architecture in Dockerfile definition.

File: powerauth-test-server/Dockerfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
-FROM tomcat:jdk11-adoptopenjdk-openj9
+FROM --platform=linux/amd64 tomcat:jdk11-adoptopenjdk-openj9
```
[the remark this answers is not in the document]
Comment (zcgandcomp) [reply powerauth-test-server/Dockerfile]: well ok

**Comment by romanstrobl** [comment]
> btw @romanstrobl I believe we do not need oss repo - unless we plan to publish it, and that we do not want it's war file. internal jfrog is fine and public maven will be found automatically IMHO

Agreed, no need to publish anything, Test server only depends on other libraries and these are accessible through standard public maven mechanism.

**Review by romanstrobl — APPROVED** [review]
Approved, however I think the OSS Sonatype Snapshot repo is redundant for builds with `-DuseInternalRepo=true`.

**Pull request merged**